### DISCUSSION TITLE
Inverse sort order for studies

### DISCUSSION THREAD

**Discussion by res80** [nroduit/Weasis#368] [Ideas]
Hello,
i was wondering if it is possible to invert the sort order of the studies.
For real life reporting, it is perfectly fine to have the most recent study on top.
However, for training purposes or for an exam, it would be better to browse through the studies of a case in chronological order, so to have the oldest study on top.
Is this perhaps already possible with some hidden trick?
Or would this be easy to add as an option?

Thanks for this amazing viewer.
Kind regards,
Ruediger

**Comment by res80** [comment]
Thanks for your fast reply.
But I didn’t mean the sort order of the Images within a series, but the sort order of the entire study in the dicom explorer.
Currently it’s most recent on top, I would like to have most recent at the Bottom.
Thanks,
Ruediger

Gesendet von Mail für Windows

Von: Nicolas Roduit
Gesendet: Freitag, 27. Jänner 2023 22:03
An: nroduit/Weasis
Cc: res80; Author
Betreff: Re: [nroduit/Weasis] Inverse sort order for studies (Discussion #368)

This is available on the main menu or on the contextual menu:

**Comment by nroduit** [comment]
Currently, there is no option for changing this behavior. However, an option can be added for the explorer.

**Comment by res80** [comment]
Thank you.

Gesendet von Mail für Windows

Von: Nicolas Roduit
Gesendet: Freitag, 27. Jänner 2023 22:11
An: nroduit/Weasis
Cc: res80; Author
Betreff: Re: [nroduit/Weasis] Inverse sort order for studies (Discussion #368)

Currently, there is no option for changing this behavior. However, an option can be added for the explorer.
